facebook/Module.md

```diff
@@ -67,161 +67,156 @@ facebook:FacebookConfiguration facebookConfig = {
 if (response is facebook:Post) {
-    p = response;
-    io:println(p);
+    io:println("Post Details: ", response);
 } else {
-    io:println(response);
+    io:println("Error: ", response);
 }
 ```
 
-The `deletePost` function deletes the post specified by the ID. The `postId` represents the ID of the post to be deleted. It returns the `True` object on success and `error` on failure.
+The `deletePost` remote function deletes the post specified by the ID. The `postId` represents the ID of the post to be deleted. It returns deletion status on success or an `error` if an error occurred.
 ```ballerina
 var response = facebookEP.deletePost(postId);
 if (response is boolean) {
-    b = response;
-    io:println(b);
+    io:println("Status: ", response);
 } else {
-    io:println(response);
+    io:println("Error: ", response);
 }
 ```
 
-The `getFriendListDetails` function used to get the User's friends who have installed the app making the query. The `userId` represents the ID of the user. It returns the `FriendList` object on success and `error` on failure.
+The `getFriendListDetails` remote function is used to get the user's friends who have installed the app making the query. The `userId` represents the ID of the user. It returns a `FriendList` object on success or an `error` if unsuccessful.
 ```ballerina
 var response = facebookEP.getFriendListDetails(userId);
 if (response is facebook:FriendList) {
-    friendList = response; 
-    io:println(friendList); 
+    io:println("Friend List: ", response);
 } else {
-     io:println(response);
+     io:println("Error: ", response);
 }
 ```
 
-The `getPageAccessTokens` function used to get the page access tokens. The `userId` represents the ID of the user. It returns the `AccessTokens` object on success and `error` on failure.
+The `getPageAccessTokens` remote function is used to get the page access tokens. The `userId` represents the ID of the user. It returns an `AccessTokens` object on success or an `error` if unsuccessful.
 ```ballerina
 var response = facebookEP.getPageAccessTokens(userId);
 if (response is facebook:AccessTokens) {
-    list = response; 
-    io:println(accessTokenList); 
+    io:println("Page Access Tokens: ", response);
 } else {
-    io:println(response);
+    io:println("Error: ", response);
 }
 ```
 
 ##### Example
 
 ```
+import ballerina/config;
+import ballerina/http;
 import ballerina/io;
 import wso2/facebook;
-import ballerina/config;
-
-public function main() {
-    string userAccessToken = config:getAsString("ACCESS_TOKEN");
-    string pageAccessToken = callMethodsWithUserToken(userAccessToken);
-    callMethodsWithPageToken(pageAccessToken);
-}
 
 facebook:FacebookConfiguration facebookConfig = {
     clientConfig:{
         auth:{
             scheme: http:OAUTH2,
-            accessToken:accessToken
+            accessToken: config:getAsString("ACCESS_TOKEN")
         }
     }
 };
 
 facebook:Client facebookclient = new(facebookConfig);
```

Comment:
Better to rename this `facebookclient` to `facebookClient` (including all the usages) as well.